src/odemis/acq/stream/_sync.py

```diff
@@ -296,13 +296,20 @@ def estimateAcquisitionTime(self):
 
                 # Count 2x as we need to go back and forth
                 tot_dist = (phy_width[0] * repetition[1] + phy_width[1]) * 2
```

Comment:
I don't think it makes sense to compute the total distance in X+Y to estimate the duration along X, and also along Y. Also, a long move is much faster than many small moves. So it should be more precise to compute the duration of every X moves and multiply by the number of moves, and same for Y.
```
pxs = self._getPixelSize()
move_time_x =  guessActuatorMoveDuration(self._sstage, "x", pxs[0]) * repetition[0]  * repetition[1]  + guessActuatorMoveDuration(self._sstage, "x", pxs[0] * repetition[0])
move_time_y =  guessActuatorMoveDuration(self._sstage, "y", pxs[1]) * repetition[1]  + guessActuatorMoveDuration(self._sstage, "y", pxs[1] * repetition[1]) 
```

Reply:
Why did you add repetition[1] in the move time computation of x in your suggestion?

Reply:
Because there will be X*Y spots. There is (roughly) as many moves in X than number of spots (actually, I think it's more like `(X-1) * Y`, so we could slightly improve that formula.

The second part of each formula corresponds to the return move at the end of the acquisition. Actually I don't think it's necessarily going back to the first spot. Maybe we should just drop that part, as it shouldn't be a large amount, and would keep it simpler.

Reply:
I improved the formula as you suggested and also renamed repetition -> rep for better readability.